.github/workflows/ci.yml

```diff
@@ -18,20 +18,20 @@ jobs:
           git clone --depth 1 https://github.com/nvim-lua/plenary.nvim ~/.local/share/nvim/site/pack/vendor/start/plenary.nvim
           git clone --depth 1 https://github.com/kyazdani42/nvim-web-devicons ~/.local/share/nvim/site/pack/vendor/start/nvim-web-devicons
           ln -s $(pwd) ~/.local/share/nvim/site/pack/vendor/start
-      - name: Get neovim v0.5.0
```

Comment:
I could otherwise recommend using the [`rhysd/action-setup-vim@v1`](https://github.com/rhysd/action-setup-vim) action. I've been using it and it's been stable so far, it'll help cut down build times as well.